docs/source/_napoleon_patch.py

```diff
@@ -16,8 +18,7 @@ def qualify_name(attr_name, klass):
 def setup(app):
     app.setup_extension('sphinx.ext.napoleon')
 
-    def _parse_attributes_section(self, section):
-        # type: (unicode) -> List[unicode]
+    def _parse_attributes_section(self, section: str) -> List[str]:
```

Comment:
> `unicode` -> `str`
I'm not sure if this is correct...:cry::bow:

Reply:
I looks OK (not so sure though).